app/_src/policies/meshtrafficpermission.md

```diff
@@ -9,12 +9,14 @@ Do **not** combine with [TrafficPermission](/docs/{{ page.version }}/policies/tr
 
 ## TargetRef support matrix
 
-| TargetRef type    | top level | to  | from |
-| ----------------- | --------- | --- | ---- |
-| Mesh              | ✅        | ❌  | ✅   |
-| MeshSubset        | ✅        | ❌  | ✅   |
-| MeshService       | ✅        | ❌  | ✅   |
-| MeshServiceSubset | ✅        | ❌  | ✅   |
+{% tabs targetRef useUrlFragment=false %}
+{% tab targetRef Sidecar %}
+| `targetRef`             | Allowed kinds                                            |
+| ----------------------- | -------------------------------------------------------- |
+| `targetRef.kind`        | `Mesh`, `MeshSubset`, `MeshService`, `MeshServiceSubset` |
+| `from[].targetRef.kind` | `Mesh`, `MeshSubset`, `MeshService`, `MeshServiceSubset` |
+{% endtab %}
```

Comment:
Worth saying that for gateways this will result in a noop?

Reply:
Mmm yeah good point.